packages/office-ui-fabric-react/src/utilities/responsiveMode/withResponsiveMode.tsx

```diff
@@ -1,21 +1,8 @@
 import * as React from 'react';
 import { findDOMNode } from 'react-dom';
-import { BaseDecorator } from './BaseDecorator';
+import { BaseDecorator } from '../decorators/BaseDecorator';
```

Comment:
I don't think this file should be moved. Because this wasn't exported from the root, people **are** importing it by deep path (@dzearing if you search ODSP code you can find *many* specific examples in active projects), and this change would break them.